fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -881,6 +881,7 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	int SIM_KMS_MAX_KEYS;
 	int ENCRYPT_PROXY_MAX_DBG_TRACE_LENGTH;
 	bool ENABLE_TLOG_ENCRYPTION;
+	bool ENABLE_STORAGE_SERVER_ENCRYPTION; // Currently only supported if Redwood is used
```

Comment:
a nit: "Currently only supported if Redwood is used" -> "Currently only Redwood engine supports encryption" maybe